<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnOnline">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>上线</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOffline">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>下线</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ipCombo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>要发送的消息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendMsg">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>发送消息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>发送文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>接受的消息</string>
    </property>
   </widget>
   <widget class="QLabel" name="ReceiveLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>351</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>364</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
